src/mbgl/renderer/painter.hpp

```diff
@@ -242,6 +242,11 @@ class Painter : private util::noncopyable {
     VertexArrayObject backgroundPatternArray;
     VertexArrayObject backgroundArray;
 
+    VertexArrayObject coveringPlaingOverdrawArray;
```

Comment:
We apply the same _alias_ technique for selecting original/overdraw VAOs, as seen in https://github.com/mapbox/mapbox-gl-native/pull/5372/files/f94b354b53ba2fc5d12aade7a82790965784afd9..d8a10ac509d5f56120eb878b5e330dcb4c7abf1b#diff-bf2327a9287221e6bae1de29dc0c3400R16 for example.